### ISSUE TITLE
Not working with Foundry's MeasureTemplate anymore.

### ISSUE THREAD

**Issue by greengaby9** [Roger92/metric-ruler-labels#37]
I recently updated to v11 and the metric labels seem to not work with the templates anymore.
I have deactivate all other modules and created fresh worlds, but nothing changes.

![image](https://github.com/Roger92/metric-ruler-labels/assets/22947110/5cb9e938-4ae3-4d2d-9d0f-9965c2367e08)

![image](https://github.com/Roger92/metric-ruler-labels/assets/22947110/d4375de7-6aa0-45b7-a514-8c1d9bd641f2)

![image](https://github.com/Roger92/metric-ruler-labels/assets/22947110/0743f673-6cba-4da0-a40f-285bb360596f)

**Comment by Roger92** [comment]
Yep known issue... i found the issue already ... but currently some private stuff going on... maybe i find an hour on the weekend to fix it.

Sorry for the issue... wanted to fix it for weeks already :/

**Comment by GoryFate** [comment]
Hi there, hope your private stuff is okay.
Do you intend to fix this problem in the near future or can you give some pointers to what is wrong so maybe I could try a bandaid fix for my installation?

**Comment by Roger92** [comment]
@GoryFate Heyho.... yah its okay enough i guess...

i have a long weekend, so i think you can expect a fix by the end of this month :) Sorry for the long delay and thx for using this module :)

**Comment by Roger92** [comment]
Alright... will do some final testing tomorrow, so that i am sure i didnt fucked up something different :D @GoryFate  So  if everything goes well you will have a new release on friday :)

**Comment by Roger92** [comment]
Found another issue that my fix caused yesterday... will fix work on that this evening... Sorry for the spam here but wanna keep you updated, since this is open for so long. 

Sry for the delay :)

**Comment by Roger92** [comment]
Aaaaaalrighty.... fixed it... will get some food now and then a new version update :) @GoryFate @greengaby9

**Comment by Roger92** [comment]
@GoryFate @greengaby9 ... Version 3.1.8 is out :) Let me know if it works for you. Again sorry for the delay. Was alot going on. :(
